pom.xml

```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>share-client</artifactId>
-  <version>7.4.0-SNAPSHOT</version>
+  <version>7.4.0</version>
```

Comment:
Please use a release version in the master branch. In the develop branch there is only a SNAPSHOT version allowed.